tests/nnfw_api/src/ValidationTestSessionCreated.cc

```diff
@@ -40,6 +40,21 @@ TEST_F(ValidationTestSessionCreated, neg_load_session_002)
       NNFW_STATUS_ERROR);
 }
 
+TEST_F(ValidationTestSessionCreated, neg_load_session_003)
+{
+  ASSERT_EQ(nnfw_load_model_from_file(_session, nullptr), NNFW_STATUS_ERROR);
+}
+
+TEST_F(ValidationTestSessionCreated, neg_load_session_004)
+{
+  // Too long path
+  const std::string RANDOM_PATH_SIZE_512(512, 'x');
+  ASSERT_EQ(
+      nnfw_load_model_from_file(
+          _session, NNPackages::get().getModelAbsolutePath(RANDOM_PATH_SIZE_512.c_str()).c_str()),
```

Comment:
I wonder why this return `NNFW_STATUS_ERROR`. 
line 50 mentions that `// Too long path`.. but its length is 512 and it does not exceed `MAX_PATH_LENGTH`?